Cargo.toml

```diff
@@ -11,15 +11,16 @@ edition = "2021"
 rust-version = "1.60"
 
 [features]
-default = ["std", "multihash/default"]
-std = ["multihash/std", "unsigned-varint/std", "alloc", "multibase/std", "serde/std"]
+default = ["std", "multihash/default", "multihash-codetable/default"]
+std = ["multihash/std", "multihash-codetable/std", "unsigned-varint/std", "alloc", "multibase/std", "serde/std"]
```

Comment:
This will add `multihash-codetable` by default. So just remove them from here.

I needed to check that myself and figured out that it's the same for the other optional dependencies as well. I guess we should fix that properly somehow. But for now at least not adding `multihash-codetable` is a good first step.